code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -43,6 +42,52 @@
 			if(prob(10))
 				M.emote(pick("twitch","blink_r","shiver"))
 
+
+/datum/reagent/hypervene
+	name = "Hypervene"
+	id = "hypervene"
+	description = "Quickly purges the body of toxin damage, radiation and all other chemicals. Causes significant pain."
+	reagent_state = LIQUID
+	color = "#19C832"
+	overdose = REAGENTS_OVERDOSE / 2
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL / 2
+	scannable = 1
+
+	on_mob_life(mob/living/M, alien)
+		. = ..()
+		if(!.) return
+		for(var/datum/reagent/R in M.reagents.reagent_list)
+			if(R != src)
+				M.reagents.remove_reagent(R.id,4 * REM)
+		M.reagent_shock_modifier += 50 //Significant pain while metabolized.
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(prob(5)) //causes vomiting
+				H.vomit()
+		M.adjustToxLoss(-4 * REM)
+		M.radiation = max(M.radiation-8*REM,0)
+
+	on_overdose(mob/living/M, alien)
+		if(alien == IS_YAUTJA) return
```

Comment:
Snowflake check that perpetuates a snowflake species when there is no reason codewise not to have it.

Reply:
@F-OS Hey man, I'm just following the current convention. I'd be more than happy to get rid of Predator exceptions.

Reply:
I understand, sorry. I'll actually submit a PR later removing those snowflake exceptions.